source/_integrations/fastdotcom.markdown

```diff
@@ -8,6 +8,7 @@ ha_release: 0.88
 ha_iot_class: Cloud Polling
 ha_codeowners:
   - '@rohankapoorcom'
+  - '@erwindouna'
```

Comment:
Should add config_entry here in the config part

Reply:
Care to elaborate on what is needed? This part is new for me.